src/pdf/base/XojPdfPage.h

```diff
@@ -37,6 +37,8 @@ class XojPdfPage {
     XojPdfPage();
     virtual ~XojPdfPage();
 
+    using Link = std::pair<XojPdfRectangle, GUri*>;
```

Comment:
Could we use a `std::pair<XojPdfRectangle, PopplerActionUri*>` here (or even a `std::pair<XojPdfRectangle, std::string>`)?
[Documentation](https://developer.gnome.org/poppler/unstable/poppler-PopplerAction.html#PopplerAction).